src/pages/LandingPages/MGCovid19/CovidLandingForm.vue

```diff
@@ -2,61 +2,42 @@
-					<label v-if="!isRecurring" for="kv-toggle-deposit">One-time contribution</label>
-				</kv-toggle>
-				<transition name="kvfastfade">
-					<kv-pill-toggle
-						v-show="isRecurring"
-						id="amount-selector"
-						:options="amountOptions"
-						:selected="amountSelected"
-						@pill-toggled="selectAmount"
-						:split-pills="true"
-					/>
-				</transition>
-				<transition name="kvfastfade">
-					<div class="custom-amount-holder" v-show="isRecurring && customAmountActive">
-						<kv-currency-input
-							id="custom-amount-input"
-							class="input-element"
-							v-model="customAmount"
-							:aria-disabled="!customAmountActive"
-							:disabled="!customAmountActive"
-						/>
-					</div>
-				</transition>
-				<transition name="kvfastfade">
-					<div class="custom-amount-holder" v-show="!isRecurring">
-						<kv-currency-input
-							id="onetime-amount-input"
-							class="input-element"
-							v-model="onetimeAmount"
-						/>
-					</div>
-				</transition>
+				<label class="deposit-options-label" for="deposit-type">Choose how often to contribute</label>
+				<kv-pill-toggle
+					id="deposit-type"
+					class="deposit-options-toggle"
+					:options="depositOptions"
+					:selected="depositSelected"
+					@pill-toggled="depositTypeChanged"
+				/>
+
+				<label class="amount-options-label">Choose an amount to contribute</label>
+				<multi-amount-selector
+					v-show="isRecurring"
+					id="amount-selector"
+					class="recurring-amount-selector"
+					:options="recurringAmountOptions"
+					:selected="recurringAmountSelection"
+					:custom-amount="recurringCustomAmount"
+					:min-custom-amount="5"
+					@pill-toggled="recurringAmountSelected"
+					@custom-amount-updated="recurringCustomAmountUpdated"
+					:split-pills="true"
+				/>
+
+				<multi-amount-selector
+					v-show="!isRecurring"
+					id="onetime-amount-selector"
+					class="onetime-amount-selector"
+					:options="onetimeAmountOptions"
+					:selected="onetimeAmountSelection"
+					:custom-amount="onetimeCustomAmount"
+					:min-custom-amount="25"
+					@pill-toggled="onetimeAmountSelected"
+					@custom-amount-updated="onetimeCustomAmountUpdated"
+					:split-pills="true"
+				/>
 			</fieldset>
-			<!-- TODO: Add inline validations once design is finalized -->
-			<!-- <fieldset class="large-4 medium-5 small-12 columns input-wrapper">
-				<label class="show-for-sr" :class="{ 'error': $v.$invalid }" :for="'amount' + componentKey">
-					Amount
-				</label>
-				<kv-currency-input :id="'amount' + componentKey" :value="amount" @input="updateAmount" />
-				<ul class="validation-errors" v-if="$v.$invalid">
-					<li v-if="!$v.amount.required">
-						Field is required
-					</li>
-					<li v-if="!$v.amount.minValue || !$v.amount.maxValue">
-						Enter an amount of $5-$10,000
-					</li>
-				</ul>
-			</fieldset> -->
 		</div>
 
 		<kv-button class="smaller" type="submit" :disabled="$v.$invalid">
```

Comment:
This line won't let the page load since $v is undefined since the validation mixin was removed. There is also no more validation on the page. This allows a user to enter an invalid amount in "Other" and proceed with the take action button. Which seems to cause some weird behavior, if the amount was too low it gets rounded up, if it was too high it seems to get truncated.

I would suggest adding validation back to this component based on the selected amount, and then you can disable the button based on that

Reply:
Yep good call. Def want to be able to disable the button.

Reply:
It's back!